app/presenters/idv/in_person/usps_form_presenter.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Idv::InPerson::UspsFormPresenter
+  include FormHelper
+  # Filtered list to remove territories that cause errors in the USPS API
+  def usps_states_territories
+    us_states_territories.reject { |_name, abbrev| %w[AA AE AP UM].include?(abbrev) }
```

Comment:
Using a constant here might help avoid some memory allocations / garbage collection when iterating the list. It might also be simpler to use [the `-` method](https://ruby-doc.org/core-3.0.1/Array.html#method-i-2D) to remove the options we don't want.

```suggestion
  include FormHelper
  
  UNSUPPORTED_STATES = %w[AA AE AP UM].freeze
  
  # Filtered list to remove territories that cause errors in the USPS API
  def usps_states_territories
    us_states_territories - UNSUPPORTED_STATES
```

Reply:
Ah, I missed the tuple shape. The use of a constant may have been an improvement on its own (local benchmarking shows ~42% performance improvement), but not a show-stopper.